### PR TITLE
Add github action to trigger container images build

### PR DESCRIPTION
**Pull request by mquhuy**
As titled.

### CODE REVIEW

File: .github/workflows/build-images-action.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: build-images-action
+on:
+  push:
+    branches:
+      - main
+      - 'release-*'
+    tags:
+      - 'v**'
```
Comment (tuminoid) [inline .github/workflows/build-images-action.yml:8]: Whats the syntax here? Regex, glob? 

This also matches `apis/v....` so probably needs start of line symbol.

Comment (mquhuy) [reply .github/workflows/build-images-action.yml:8]: It's said to use glob:
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-branches-for-pull-request-events

Comment (mquhuy) [reply .github/workflows/build-images-action.yml:8]: Thanks for the clarification. I've an impression that I saw v*/* pattern somewhere, but if that's not what we want, then `v*` should suit better.

**Review by tuminoid — COMMENTED** [review]
LGTM now

**Comment by tuminoid** [comment]
@zaneb @elfosardo @dtantsur
can we get an LGTM  and approve here?

**Comment by tuminoid** [comment]
Build on Quay side has been disabled. Container image will be built by GH action + Jenkins from now on.

**Pull request merged**